src/base/interfaces.ts

```diff
@@ -430,11 +447,29 @@ export interface IViewPlugin {
   factory(context: IViewContext, selection: ISelection, parent: HTMLElement, options?: any): IView;
 }
 
-export interface IViewWrapperDump {
-  hash: string;
-  plugin: string;
-  dumpReference: number;
-  parameters: object;
+export interface IVisynViewProps<Desc extends IVisynViewPlugin, Param extends Record<string, any>> {
```

Comment:
I think this should be 
```suggestion
export interface IVisynViewProps<Desc extends IVisynViewPluginDesc, Param extends Record<string, any>> {
```
Since the  `IVisynViewPluginDesc` interface should be extendable and not the `IVisynViewPlugin `
Also in my components i would have to do like `props.desc.desc` to access any values